stats.rst

```diff
@@ -0,0 +1,209 @@
+.. _stats:
+
+=================
+Stats Quick Start
+=================
+
+A quick guide to getting the internal stats collection working in flux.
+This guide assumes you have already built a flux binary; if you have not see :ref:`quickstart`.
+
+.. _brubeck:
+
+------------------
+Installing Brubeck
+------------------
+
+Brubeck is a statsd-compatible metric aggregator that sits in the middle of the stack between flux and a frontend tool like Graphite.
+
+.. code-block:: console
+
+  $ git clone git@github.com:lukepalmer/brubeck.git
+  # or git clone  https://github.com/lukepalmer/brubeck.git
+  $ cd brubeck
+  $ ./script/bootstrap
+  $ cp config.default.json.example config.default.json
+  $ make test
+
+If all of the tests pass then it's time to move on to the frontend and third piece of the stack, Graphite.
+
+-------------------
+Installing Graphite
+-------------------
+
+Install Graphite from your local package manager.
```

Comment:
Should it also be mentioned that you can run graphite in docker? Or does that not work for some reason?

Reply:
All of the docker images I've found with Graphite also contain statsd which would work fine unless someone is using the `flux_stats_count()`

Reply:
You could just run brubeck on a different port or not export the statsd udp port on the docker line. However, I can experiment with that and update the docs down the road.

Reply:
For now should I add a note about docker at the top?

Reply:
You could mention it -- however making the config changes you mention below in the docker image may be tricky.

Later on, maybe we'll make our own `fluxrm/graphite` docker image with your suggested configuration and a copy of brubeck instead of statsd -- but don't worry about it too much for now.

Reply:
Interesting, I just learned that the latest `graphiteapp/graphite-statsd` docker image has brubeck installed: 

https://github.com/graphite-project/docker-graphite-statsd#brubeck